src/pages/lock-exception/LockException.js

```diff
@@ -73,7 +73,11 @@ const LockExceptionsTable = ({ columns, rows, onRemoveLockException }) => (
                         <TableCell key={column}>{row[column]}</TableCell>
                     ))}
                     <TableCell>
-                        <Button small secondary onClick={onRemoveLockException}>
+                        <Button
+                            small
+                            secondary
+                            onClick={onRemoveLockException.bind(null, row)}
```

Comment:
I think this was an actual regression: there wasn't a lock-exception object being passed to `onRemoveLockException`.

Reply:
Yep - this must have been from the migration from `d2-ui`'s `DataTable` to the `@dhis2/ui` `Table`